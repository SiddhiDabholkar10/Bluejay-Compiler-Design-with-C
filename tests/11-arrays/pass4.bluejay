var variable int := 2;
arr[1] int := 1;
return arr[5*2-1];
return arr[6*12+7*variable];

