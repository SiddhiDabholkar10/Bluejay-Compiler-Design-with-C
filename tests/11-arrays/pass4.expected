Variable declared "variable" type int
Array written "arr" type int
Array read "arr" type int
Array read "arr" type int
